src/sync/chase_lev.rs

```diff
@@ -269,9 +283,11 @@ impl<T> Drop for Deque<T> {
     fn drop(&mut self) {
         let guard = epoch::pin();
 
-        let t = self.top.load(SeqCst);
-        let b = self.bottom.load(SeqCst);
-        let a = self.array.swap(None, SeqCst, &guard).unwrap();
+        // exclusive access here means that we only need to enforce sequential consistency once.
+        fence(SeqCst);
```

Comment:
Ah, I hadn't considered that `Arc` also enforced the sequential consistency. I had figured I should add it just to be safe, but it seems it can be removed now.